### PR TITLE
add EPICS module support busy

### PR DESCRIPTION
**Pull request by agaget**
add EPICS module support busy

### CODE REVIEW

File: pkgs/epnix/support/busy/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    repo = "busy";
+    rev = "R${version}";
+    sha256 = "sha256-mSzFLj42iXkyWGWaxplfLehoQcULLpf745trYMd1XT4=";
+  };
+
+  patches = [./fix-release.patch];
+
+  meta = {
+    description = "APS BCDA synApps module: busy";
+    homepage = "APS BCDA synApps module: busy";
```
Comment (minijackson) [inline pkgs/epnix/support/busy/default.nix:29]: `homepage` should be a link to the home page of the project

File: pkgs/epnix/support/busy/default.nix [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  src = fetchFromGitHub {
+    owner = "epics-modules";
+    repo = "busy";
+    rev = "R${version}";
+    sha256 = "sha256-mSzFLj42iXkyWGWaxplfLehoQcULLpf745trYMd1XT4=";
+  };
+
+  patches = [./fix-release.patch];
+
```
Comment (minijackson) [inline pkgs/epnix/support/busy/default.nix:26]:
```suggestion
  src = fetchFromGitHub {
    owner = "epics-modules";
    repo = "busy";
    rev = "R${version}";
    sha256 = "sha256-mSzFLj42iXkyWGWaxplfLehoQcULLpf745trYMd1XT4=";
  };

  patches = [./fix-release.patch];

  buildInputs = with epnix.support; [calc asyn autosave];

```

**Review by minijackson — COMMENTED** [review]
Thanks! Is there any way you can add an integration check for this module?

**Comment by minijackson** [comment]
Can you write or expand the autosave test so that it test the `configMenu.db` with its `busy` record then?

**Comment by agaget** [comment]
> Can you write or expand the autosave test so that it test the `configMenu.db` with its `busy` record then?

I will not be able to do that before this summer to not block [Synthetica9](https://github.com/Synthetica9) if he needs it, can you review/merge this part and make me an issue @minijackson ?

**Review by minijackson — APPROVED** [review]

**Pull request merged**